### ISSUE TITLE
#[path] attribute clashes with built-in attribute

### ISSUE THREAD

**Issue by cosmicspacedragon** [bschwind/app-route#1]
I tried compiling the example in the readme, but `rustc` complained and spat out this message:
```
error: attribute must be of the form `#[path = "file"]`
  --> src\main.rs:15:1
   |
15 | #[path("/groups/:group_id/users")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
This seems to be caused by the build-in [`path` attribute](https://doc.rust-lang.org/reference/attributes.html#path).

I'm using the latest `rustc` (1.34.0), so I'm guessing you're using an older compiler, @bschwind.

**Comment by bschwind** [comment]
Interesting, I thought I was on the latest rustc version, I'll check again when I'm near a computer.

**Comment by bschwind** [comment]
You're right, I was on 1.33. I'll rename it to something like `route` instead and publish a new version. I had no idea you could specify paths like that for modules.

**Comment by bschwind** [comment]
I published version `0.2.1` (I screwed up a README symlink in `0.2.0`) which changes the `path` attribute to `route`. I ran it with rust 1.34, let me know if you have any other issues with it!
